apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -175,6 +176,40 @@ export const DEFAULT_BLOCKS: Record<
     title: "Rick Astley - Never Gonna Give You Up",
     url: "https://www.youtube.com/embed/dQw4w9WgXcQ?si=ggGGn4uvFWAIelWD",
   },
+  // TODO: Replace with actual working API endpoint
+  [DYNAMIC_STATISTICS_TYPE]: {
+    type: `${DYNAMIC_STATISTICS_TYPE}`,
+    apiEndpoint: "https://jsonplaceholder.com/muis_prayers_time",
```

Comment:
in actual case for this would be a link to a csv for MUIS right?

Reply:
it will be a link to a URL that returns a JSON, which means we have to convert the CSV to JSON before uploading to S3

reason being 
1. unsure of how CSV parsing is supported on older browsers
2. using JSON allow us to reuse this block in the future for other API endpoints